client/src/components/records.tsx

```diff
@@ -1,52 +1,78 @@
 import { Table, TableCaption, TableRow, TableCell, TableHead, TableBody } from '@cmsgov/design-system';
+import axios from 'axios';
+import { AnyRecord } from 'dns';
+import { useEffect, useState } from 'react';
+import dataviewer from './dataviewer';
+
+export type EOBRecord = {
+    id: string,
+    code: string,
+    display: string,
+    amount: number
+}
+
+
+export default function Records({ }) {
+    const [records, setRecords] = useState<EOBRecord[]>([]);
+    /*
+    * DEVELOPER NOTES:
+    *  Here we are parsing through the different PDE Claim records
+    * for the user/beneficiary.  We have hard coded certain pieces of this data. (ie...item[0])
+    * You will want to find a method of parsing the FHIR JSON response to get the data
+    * you need for your application.  Don't forget to use a 'Discriminator' to determine
+    * which item within a list you want to get data from.
+    * 
+    * ie.  You are interested in getting all medications NDC codes you would use the following criteria/discriminator
+    * resource.item[N].coding[N].code WHERE resource.item[N].coding[N].system = "http://hl7.org/fhir/sid/ndc"
+    * 
+    */
+    useEffect(() => {
+        fetch('/api/data/benefit')
+            .then(res => {
+                return res.json();
+            }).then(eobData => {
+                const records: EOBRecord[] = eobData.entry.map((resourceData: any) => {
+                    const resource = resourceData.resource;
+                    return {
+                        id: resource.id,
+                        code: resource.item[0]?.productOrService?.coding[0]?.code,
+                        display: resource.item[0]?.productOrService?.coding[0]?.display || 'Unknown Medication',
+                        amount: resource.item[0]?.adjudication[7]?.amount?.value
+                    }
+                });
+                console.log(records);
```

Comment:
We could probably take this console out.

Reply:
Removed.  I found other instances where we are using the console.log and removed them where it made sense as well.  Thanks!